app/components/account/TokenExtensionsCard.tsx

```diff
@@ -0,0 +1,283 @@
+
+async function fetchTokenInfo([_, address, cluster, url]: ['get-token-info', string, Cluster, string]) {
+    return await getTokenInfo(new PublicKey(address), cluster, url);
+}
+
+export function TokenExtensionsCard({ address }: { address: string }) {
+    const { cluster, url } = useCluster();
+    const refresh = useFetchAccountInfo();
+    const mintInfo = useMintAccountInfo(address);
+    const swrKey = useMemo(() => getTokenInfoSwrKey(address, cluster, url), [address, cluster, url]);
+    const { data: tokenInfo } = useSWR(swrKey, fetchTokenInfo);
+
+    if (!mintInfo || !mintInfo.extensions) return null;
+
+    const extensions = populateTokenExtensions(mintInfo.extensions);
+
+    return (
+        <div className="card">
+            <AccountHeader title="Extensions" refresh={() => refresh(new PublicKey(address), 'parsed')} />
+            <div className="card-body p-0">
+                <TokenExtensionsSection
+                    decimals={mintInfo.decimals}
+                    extensions={mintInfo.extensions}
+                    parsedExtensions={extensions}
+                    symbol={tokenInfo?.symbol}
+                />
+            </div>
+        </div>
+    );
+}
+
+function populateTokenExtensions(extensions: TokenExtension[]): ParsedTokenExtension[] {
+    const result = extensions.reduce((acc, { extension, state }) => {
+        const data = populatePartialParsedTokenExtension(extension);
+        acc.set(extension, {
+            ...data,
+            extension: extension,
+            parsed: state,
+        });
+
+        return acc;
+    }, new Map<string, ParsedTokenExtension>());
+
+    return Array.from(result.values());
+}
+
+function populateSolanaDevelopersLink(component: string) {
+    return `https://solana.com/developers/guides/token-extensions/${component}`;
+}
+
+function populatePartialParsedTokenExtension(
+    extension: TokenExtensionType
+): Omit<ParsedTokenExtension, 'parsed' | 'extension'> {
+    function populateExternalLinks(url: string) {
+        return [{ label: 'Docs', url }];
+    }
+
+    /**
+     * Sample.
+     * TODO: To be removed upon editing all the details for each extension
+        description: 'Delegates permanent authority to a specific address that can transfer or burn tokens from any account holding this token, providing centralized administrative control over the token ecosystem.',
+        externalLinks: populateExternalLinks('https://solana.com/developers/guides/token-extensions/permanent-delegate'),
+        name: 'Permanent Delegate',
+        status: 'active',
+        tooltip: 'Delegates permanent authority to a specific address that can transfer or burn tokens from any account holding this token, providing centralized administrative control over the token ecosystem.',
+     */
+    switch (extension) {
+        case 'transferFeeAmount': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('transfer-fee')),
```

Comment:
description: "Every transfer sets aside a fee in the recipient's Token Account that can only be withdrawn by the Withdraw Authority"

Reply:
fixed